lib/src/specs/method.dart

```diff
@@ -180,6 +180,12 @@ abstract class Parameter extends Object
   /// This is only valid on constructors;
   bool get toThis;
 
+  /// Whether this parameter should be passed to super\
+  /// constructor (i.e. `super.`).
+  ///
+  /// This is only valid on constructors;
+  bool get toSuper;
```

Comment:
A bit of a weird name, but I suppose it is consistent with `toThis`?

Reply:
Yep, I've just copied the naming scheme from `toThis`.